packages/server/src/service/contact/index.ts

```diff
@@ -53,7 +53,27 @@ export const toContact = async (locale: Locale): Promise<ContactData | null> =>
 
   for (const key of keys(locale as Locale<AvailableState>)) {
     const result = await getContact(state, key)
-    if (result) return result
+    if (result) {
+      if (contactOverride[result.key]) {
+        const override = {...contactOverride[result.key]}
+        if (override.emails) {
```

Comment:
I would just do `{**result, **override}` to keep it simple for now since we're just replacing values.